build.gradle.kts

```diff
@@ -30,7 +30,14 @@ group = "com.splunk"
 subprojects {
   version = rootProject.version
 
-  apply(plugin = "splunk.java-conventions")
-  apply(plugin = "splunk.spotless-conventions")
+  if (this.name != "dependencyManagement") {
+    apply(plugin = "splunk.java-conventions")
+    apply(plugin = "splunk.spotless-conventions")
+    dependencies {
+      for(conf in configurations) {
+        add(conf.name, platform(project(":dependencyManagement")))
+      }
```

Comment:
Nit: WDYT about putting it into the `java-conventions` file?

Reply:
Oh that's a good idea I think. I do expect any small change to just break all of the gradle build completely, but let's try it...

Reply:
Looks like that worked out. Keeps some of the muck out of the top-level build script so I think I like it. 🤘🏻